index.html

```diff
@@ -50,5 +50,155 @@ <h2 class="Description1">I'am a Software Developer</h2>
             </li>
         </ul>
 </section>
+<section class="grid-container">
+  <div class="main-div">
+    <div class="top-div">
+      <h2 id="MyRecentWorks">My Recent Works</h2>
+      <img src="images/Vector.png" alt="horizontal-line" id="hr-line">
+      <img src="images/Img Placeholder.png" alt="yoga image" id="yoga-image">
+    </div>
+    <div class="butm-div">
+      <h2>Multi-Post Stories</h2>
+      <p>
+        A daily selection of privately personalized reads; no accounts or 
+        sign-ups required. has been the industry's standard dummy text ever 
+        since the 1500s, when an unknown printer took a standard dummy text.
+      </p>
+      <ul class="list1">
+        <li>CSS</li>
+        <li>HTML</li>
+        <li>Bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <a href="#" class="button">See project</a>
+    </div>
+  </div>
+</section>
+<section class="project_list">
+  <div class="project2">
+    <h3>Profesional Art <br>Printing Data</h3>
+  <p>
+    A daily selection of privately personalized reads; no accounts or sign-ups 
+    required. Has been the industry's standard.
+  </p>
+  <ul class="list2">
+    <li>HTML</li>
+    <li>Bootstrap</li>
+    <li>Ruby</li>
+  </ul>
+  <a href="#" class="button">See project</a>
+  </div>
+  <div class="project2">
+    <h3>Profesional Art <br>Printing Data</h3>
+  <p>
+    A daily selection of privately personalized reads; no accounts or sign-ups 
```

Comment:
- [ ] Kindly make sure your project list is visible, as you may notice on the UI, Only the `see project` button can be seen for now. Please ensure the user can see the project for a better user experience.
### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/229281294-74e2ca31-f516-4768-b44e-acb62ca3f9f7.png)
### Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/229283937-81d45105-65e1-4592-96c5-0e56af90aa15.png)

Reply:
@christianbiring1 I have committed the changes

Reply:
@sahlminkok , you can go ahead and submit again, a reviewer will review the changes.

Good luck!